creator-node/src/services/stateMachineManager/stateMonitoring/findSyncRequests.jobProcessor.js

```diff
@@ -210,7 +241,17 @@ const _findSyncsForUser = (
         errors.push(
           `Error getting new or existing sync request for user ${wallet} and secondary ${secondary} - ${e.message}`
         )
+        continue
       }
+    } else if (syncMode === SYNC_MODES.SyncPrimaryFromSecondary) {
+      /**
+       * TODO - currently just logs as placeholder
+       * 1. await syncFromSecondary()
+       * 2. issue sync to secondary with forceResync = true
+       */
+      logger.info(
+        `[findSyncRequests][_findSyncsForUser][SyncPrimaryFromSecondary = true][SyncType = ${SyncType.Recurring}] wallet ${wallet} secondary ${secondary} Clocks: [${primaryClock},${secondaryClock}] Files hashes: [${primaryFilesHash},${secondaryFilesHash}]`
+      )
     }
   }
 
```

Comment:
it would be great for debugging purposes if we returned syncMode here

Reply:
just realized this is in each sync for each user so nvm we shouldn't need to do anything else since you can tell from the `primaryEndpoint` and `secondaryEndpoint` properties of each object in the `syncReqsToEnqueue` array

Reply:
yeah we would but having that syncMode property would definitely make it much easier to see whats happening, i added it - i think its a good idea

Reply:
great callout - i think it would prob be nice to expose syncMode everywhere. i tried to add it but it gets a bit complex, will do as a follow up